src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -135,7 +136,8 @@ public IActionResult GetFunctionStatus(string name, [FromServices] IScriptJobHos
             {
                 // if we don't have any errors registered, make sure the function exists
                 // before returning empty errors
-                FunctionDescriptor function = scriptHost.Functions.FirstOrDefault(p => p.Name.ToLowerInvariant() == name.ToLowerInvariant());
+                var result = await _functionsManager.GetFunctionsMetadata(Request, includeProxies: true);
```

Comment:
Above, I've removed the RequiresRunningHost annotation completely, and moved this API to use the functions manager which reads from files and doesn't require a running host. When in offline mode, the offline host is completely empty - no functions will have been loaded. This makes this status api more consistent with the List/Get function apis.

Reply:
We may want to have the `scriptHost` optional here and deal with it possibly being `null`. This would handle cases where this API may be called before the child container has that service.

Reply:
I'll handle this in a follow up.